parts/kubernetesagentcustomdata.yml

```diff
@@ -113,13 +116,13 @@ write_files:
       --volume=/srv/kubernetes/:/srv/kubernetes/:ro \
         {{{kubernetesHyperkubeSpec}}} \
           /hyperkube kubelet \
-            --api-servers="https://{{{masterInternalLbIp}}}:443" \
```

Comment:
It's a deprecated flag. The `kubeconfig` file already has the masters location inside, so this flag was deprecated. The `require-kubeconfig` flag is required for a reason I don't 100% fully understand, but I think it's a temporary thing and will go away after 1.6.